vocabs/oada/isoblue/index.js

```diff
@@ -0,0 +1,72 @@
+var libvocab = require('../../../lib/vocab')('oada');
+// Only importing the essentials, more complex definitions may need more imports
+// similar to the main oada index.js this is based off of
+const {register, override, enumSchema} = libvocab;
+
+// Custom location schema
+register('lat', override('latitude', {
+    description: 'Short form of latitude term, otherwise the same. ' +
+                 'Used in ISOBlue CAN data.'
+}));
+  
+register('lng', override('longitude', {
+    description: 'Short form of longitude term, otherwise the same. ' +
+                 'Used in ISOBlue CAN data.'
+}));
+
+
+// Custom CAN schema
+register('is_error_frame',  {
+    description: 'Error frame flag for can frame',
+    type: Boolean,
+});
+
+register('is_extended_frame', {
+    description: 'Extended frame flag for can frame',
+    type: Boolean,
+});
+
+register('arbitration_id', {
+    description: 'CAN ID portion of extended can frame',
+    type: Number,
+});
+
+register('dlc',  {
+    description: 'Data Length Code portion of extended can frame',
+    type: Number,
+});
+
+register('payload', {
+    description: 'String representation can frame payload in base64',
+    type: String,
+});
+
+register('is_remote_frame', {
+    description: 'Remote frame flag from can frame',
+    type: Boolean,
+});
+
+// Custom heartbeat schema
+register('backlog', override('generic-data-value', {
+    description: "Length of backlog in computer systems. Used in ISOBlue"+
+                 " to send amount of unsent messages",
+    properties: {
+        'units': enumSchema( [ 'count' ] ),
+    },

```

Comment:
This vocab does not match the example you gave that used backlog.  This vocab schema is correct, the examples are wrong.  This schema makes `backlog` an object with possible keys of value and units